docusaurus/docs/reference/config-types/host.v2.md

```diff
@@ -0,0 +1,291 @@
+---
+sidebar_label: host.v2
+sidebar_position: 20
+---
+
+# The `Host.v2` Config Type
+
+The `host.v2` configuration type defines how edge routers or tunnelers should create terminators and
+health checks for the application endpoints that provide a service.
+
+## Terminators
+
+Each config can define one or more terminators. If a service is hosted by multiple servers, the
+configuration can have a terminator defined for each hosting server.
+
+For services with defined endpoints, the following three attributes are required:
+
+* `address` - an IP or DNS address to connect to
+* `port` - the network port to connect to
+* `protocol` - tcp or udp
+
+**Examples**
+
+This service is hosted by one application server.
+
+```json
+{
+  "terminators": [
+    {
+      "address": "192.168.100.1",
+      "port": 80,
+      "protocol": "tcp"
+    }
+  ]
+}
+```
+
+This service is hosted by two application servers.
+
+```json
+{
+  "terminators": [
+    {
+      "address": "192.168.100.1",
+      "port": 80,
+      "protocol": "tcp"
+    },
+    {
+      "address": "192.168.100.2",
+      "port": 80,
+      "protocol": "tcp"
+    }
+  ]
+}
+```
+
+Services that are forwarding traffic from an OpenZiti tunneling proxy may use the following
+properties to indicate what should be forwarded:
+
+* `forwardProtocol` - flag indicating that the protocol of the forwarded connection is to be used
+    * Must be the value `true`
```

Comment:
> Must be the value `true`

reads awkwardly. Maybe "Can only be set to true" or "If set, the value must only be set to true" and then probably not in bulletted form?

Reply:
fixed